Packs/FreshworksFreshservice/Integrations/FreshworksFreshservice/FreshworksFreshservice.yml

```diff
@@ -0,0 +1,4191 @@
+      isArray: false
+      name: subject
+      required: true
+    - auto: PREDEFINED
+      description: The channel through which the Ticket was created. The default value is 'Portal'. (Email=1, Portal=2, Phone=3, Chat=4, Feedback widget=5, Yammer=6, AWS Cloudwatch=7, Pagerduty=8, Walkup=9, Slack=10).
+      isArray: false
+      name: source
+      predefined:
+      - Email
+      - Portal
+      - Phone
+      - Chat
+      - Feedback widget
+      - Yammer
+      - AWS Cloudwatch
+      - Pagerduty
+      - Walkup
+      - Slack
+      required: false
+    - description: Tags that have been associated with the ticket.
+      isArray: true
+      name: tags
+      required: false
+    - description: Department ID of the requester. Use freshservice-department-list to get the department ID.
+      isArray: false
+      name: department_id
+      required: false
+    - auto: PREDEFINED
+      description: Ticket category.
+      isArray: false
+      name: category
+      predefined:
+      - Hardware
+      - Software
+      - Network
+      - Office Applications
+      - Talent Management
+      - Travel
+      - Employee Records and Documents
+      - Office Furniture
+      - Office Equipment
+      - Employee Benefits
+      - Employee Onboarding/Offboarding
+      - Employee Relations
+      - Workplace Access and Security
+      - Building and Grounds Maintenance
+      - Vendor Document Review
+      - Payroll
+      - Vendor Payment
+      - Customer Payment
+      - Reimbursements and Advances
+      - Legal Document Creation
+      - Legal Review - Vendor Documents
+      - Legal Review - Customer Documents
+      - Other
+      required: false
+    - description: Ticket subcategory. Note that each category has a different predefined subcategory.
+      isArray: false
+      name: sub_category
+      required: false
+    - description: Assets that have to be associated with the ticket (the asset display_id).
+      isArray: false
+      name: assets
+      required: false
+    - auto: PREDEFINED
+      description: Ticket urgency.
+      isArray: false
+      name: urgency
+      predefined:
+      - Low
+      - Medium
+      - High
+      required: false
+    - auto: PREDEFINED
+      description: Ticket impact.
+      isArray: false
+      name: impact
+      predefined:
+      - Low
+      - Medium
+      - High
+      required: false
+    - description: The problem that needs to be associated with the Ticket  (use freshservice-problem-list to get the problem ID).
+      isArray: false
+      name: problem
+      required: false
+    - description: Change causing the Ticket that needs to be associated with the Ticket (use freshservice-change-list to get the change ID).
+      isArray: false
+      name: change_initiating_ticket
+      required: false
+    - description: Change needed for the Ticket to be fixed that needs to be associated with the Ticket (use freshservice-change-list to get the change ID).
```

Comment:
This description is not clear. I am not sure what it is trying to say.

Reply:
the change (ID) that needed for the Ticket to be fixed (so the change ID needs to be associated with the Ticket).